### ISSUE TITLE
Configure API GateWay Production Endpoint

### ISSUE THREAD

**Issue by ZakBrinlee** [ActoKids/api#37]
Configure API Gateway with domains and SSL for all endpoints for the Production environment. 

**STEPS FOR SPRINT:**
- Teammate from the DevOps for this task: Lyndon Pereira
- Steps Summarized from Jon's [wiki](https://github.com/ActoKids/api/wiki/API-accessability-via-HTTPS-custom-domain) to create a custom domain for API Gateway API.  Very good wiki as well.
- Hosted Zone in AWS Route 53 was already created.
- I used AWS Certificate Manager to request a certificate.
- I added the custom domain to API Gateway and used regional API deployment option.
- I created a test Lambda in production region, Oregon, to show the prod endoint is working as you can test below.
- Lyndon gave me permissions for all of these: Cert. Manager, API Gateway, Lambda so I could complete this sprint.  See bug links below for more info on this.  In short this was because production is in Oregon and the api-team has been in Ohio so I lacked needed permissions for this sprint.
- **Prod Endpoint DEMO** ->   [https://api.2edusite.com/v1/events](https://api.2edusite.com/v1/events)
- One thing to note, the endpoint only works with https and not http

**TIME SPENT**
- Estimated Time: 8 hours
- Time Spent: 7 hours

**BUGS CREATED IN DEVOPS REPO**
- https://github.com/ActoKids/devops/issues/34
- https://github.com/ActoKids/devops/issues/35

**WIKI**: [click here](https://github.com/ActoKids/api/wiki/Configure-API-GateWay-Production-Endpoint)

**Bug Assigned to Me**
[bug](https://github.com/ActoKids/api/issues/31)

**Comment by ZakBrinlee** [comment]
**Review by Zak Brinlee:**
Production endpoint URL is configured. I was able to clearly follow Travis' steps and his Wiki. 

**Testing:**
I tested Travis' task by going to the endpoint he created and confirming it is the accurate domain endpoint. 

* **Prod Endpoint DEMO** ->   https://api.2edusite.com/v1/events

**Notes:**
As Travis pointed out, this does not work with HTTP, only HTTPS. Same situation with the DEV environment.

**Comment by Enshaedn** [comment]
**Review by Alex Clontz:**
Production endpoint URL is configured and working. 

**Testing:**
Tested the endpoint and confirmed it is working
> [https://api.2edusite.com/v1/events](https://api.2edusite.com/v1/events)

![image](https://user-images.githubusercontent.com/23545971/53689780-2ae70680-3d12-11e9-8972-f5c7622b8a51.png)

**Notes:**
Must use HTTPS - true for every regional setup. Certificate requires HTTPS and there is no re-route setup for HTTP for regional. Also, while I understand the reason for creating everything manually to try to test it I think it isn't the best solution for the prod and dev regions as it is giving our team access they shouldn't have and circumventing the TravisCI pipeline. Therefore I created a [ticket/bug](https://github.com/ActoKids/devops/issues/37) in the DevOps repo noting that TravisCI isn't doing what it is supposed to be doing.
